nbs/11_clean.ipynb

```diff
@@ -172,6 +180,34 @@
     "test_eq(nb['metadata'], {'kernelspec': 'some_spec', 'jekyll': 'some_meta'})"
    ]
   },
+  {
```

Comment:
Remember that tests are also documentation in nbdev projects. There's too much code here for people to easily see what's going on. Try to strip it down to something minimal which a reader could understand at a glance. If you really need a test that isn't suitable for documentation, add `#|hide` to the top of your cell (but avoid if possible)